LOGGING_template.toml

```diff
@@ -0,0 +1,68 @@
+### logging/warning template for inkstitch
+### format: toml
+### enable config file in DEBUG.toml: log_config_file
+###
+### warnings.simplefilter(action), default: "default"
+### - possible values: "error", "ignore", "always", "default", "module", "once"
+warnings_action = "default"
+
+### logging.captureWarnings()  default: true
+### - possible values: true, false
+warnings_capture = true
+
+### mandatory, must be an integer and 1
+version = 1
+
+### disable loggers not specified here true/false - may be overrided by root logger
+disable_existing_loggers = false
```

Comment:
As far as I can tell, `disable_existing_loggers` doesn't actually do anything.  I can't see where we check that value and do the actual disabling of loggers?

Reply:
It wasn't clear to me either what it was for, so I ended up looking into the Python source code. It serves to disable loggers that were activated before using the configuration from this config file. See `_handle_existing_loggers(existing, child_loggers, disable_existing)` in `python/logging/config.py`.
I've corrected the note.